setup.cfg

```diff
@@ -120,5 +120,7 @@ ignore_missing_imports = True
 ignore_missing_imports = True
 [mypy-libqtile.core._ffi_xcursors]
 ignore_missing_imports = True
+[mypy-libqtile.widget._pulse_audio]
+ignore_missing_imports = True
```

Comment:
I don't think this is quite right; it really is complaining about a bug, you're changing the signature of InLoopPollText; seems like you can just drop that from the inheritance chain all together.

Reply:
well, at least now mypy doesnt complain about import error of that module

Reply:
Yes, but I don't think we want to merge it like this.

Reply:
aaaand its failing now